pychromecast/socket_client.py

```diff
@@ -649,16 +650,16 @@ def _check_connection(self) -> bool:
         # check if connection is expired
         reset = False
         if self._force_recon:
-            self.logger.warning(
-                "[%s(%s):%s] Error communicating with socket, resetting connection",
+            self.logger.debug(
```

Comment:
Debug is too low level, let's do info

Reply:
All places that set force_recon have already logged an error

Reply:
Also this log is rather useless, below low where it is increased would be more usefull.
```
ogger: pychromecast.socket_client
Källa: /usr/local/lib/python3.12/site-packages/pychromecast/socket_client.py:655
Inträffade först: 25 augusti 2024 kl. 02:00:06 (41 händelser)
Senast loggade: 19:26:34

[Announce Group(192.168.16.29):32096] Error communicating with socket, resetting connection
[Hela Huset(192.168.16.27):32053] Error communicating with socket, resetting connection
[Home Kök(192.168.16.29):8009] Error communicating with socket, resetting connection
[Cast Bibliotek(192.168.16.152):8009] Error communicating with socket, resetting connection
[65OLED855/12(192.168.16.36):8009] Error communicating with socket, resetting connection
```